### PR TITLE
Revert "Removed custom messages from saved search templates [PD-827]"

### PR DESCRIPTION
**Pull request by almathew**
Reverts DirectEmployers/MyJobs#1143

### CODE REVIEW

**Comment by aspidites** [comment]
Why?

File: mysearches/migrations/0003_auto__del_field_savedsearch_custom_message__del_field_partnersavedsear.py [hunk 1]
```diff
@@ -1,420 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (tdphillips) [inline mysearches/migrations/0003_auto__del_field_savedsearch_custom_message__del_field_partnersavedsear.py]: If this is already on production, should a migration be added to re-add that field?

**Comment by aspidites** [comment]
@tdphillips I just merged #1143 half an hour ago

**Comment by almathew** [comment]
I misunderstood the ticket apparently.

And then I poorly explained what I thought was the ticket when I clarified it with Jason.

**Pull request merged**